test/Microsoft.DotNet.Tools.Tests.Utilities/NonWindowsOnlyFactAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.DotNet.PlatformAbstractions;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public class NonWindowsOnlyFactAttribute : FactAttribute
+    {
+        public NonWindowsOnlyFactAttribute()
+        {
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Windows)
+            {
+                this.Skip = "This test requires windows to run";
```

Comment:
Also, I'd call it `Windows`. The lower case implies that the test requires a portal through a wall!